correct.csv

```diff
@@ -0,0 +1,11 @@
+type,fruit,quantity

```

Comment:
let's remove the files that are located in the root project folder, and not in resources